### ISSUE TITLE
Update for 2018!

### ISSUE THREAD

**Issue by rafkuhn** [lemongrabs/get-here#20]
Hi y’all! Any way you guys can refresh the data so that trips for 2018 can populate?

**Comment by zane** [comment]
Sure thing! I'm at a work intensive in Boston all this week. Hopefully I'll be able to get to this next weekend.

**Comment by rafkuhn** [comment]
How hard would it be to turn this into a downloadable app? The competing ferry app website is down, so now’s our chance!

**Comment by zane** [comment]
Quite hard. Among other reasons I've been out of the mobile game for a while. It would take significant time and effort for me to ramp up on the current state of the field.

**Comment by rafkuhn** [comment]
Oh boy!

**Comment by zane** [comment]
Yeah. You'd probably be better off hiring contractors. I could give you some referrals if you want to go that route.

**Comment by rafkuhn** [comment]
👋🏻✋🏻👋🏻 😺

**Comment by zane** [comment]
I'm sorry @rafkuhn, I haven't had time to do this just yet. 😓

**Comment by rafkuhn** [comment]
I figured! Just “bumping” 😺

**Comment by zane** [comment]
FYI, the schedule is [here](https://github.com/lemongrabs/get-here/blob/master/src/get_here/ferry.clj#L51-L209). Does it make sense to you? If so if you send me an updated version I can swap it in! Otherwise @oldwestaction and I will get to it when we can. ❤️

**Comment by zane** [comment]
P.S. @rafkuhn I hope the wedding is fun tonight! 🍾

**Comment by rafkuhn** [comment]
I can try!
